distmat/wishart.go

```diff
@@ -0,0 +1,208 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distmat
+
+import (
+	"math"
+	"math/rand"
+	"sync"
+
+	"github.com/gonum/mathext"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+	"github.com/gonum/stat/distuv"
+)
+
+// Wishart is a distribution over d×d positive symmetric definite matrices. It
+// is parametrized by a scalar degrees of freedom parameter ν and a d×d positive
+// definite matrix V.
+//
+// The Wishart PDF is given by
+//  p(X) = [|X|^((ν-d-1)/2) * exp(-tr(V^-1 * X)/2)] / [2^(n*d/2) * |V|^(n/2) * Γ(d, ν/2)]
+// where tr is the trace and Γ_p is the multivariate gamma function.
```

Comment:
I'll add it to the comment, but X is a positive definite matrix of the size of V

I'll fix Gamma_p. Wikipedia writes it that way to disambiguate from the non-multivariate Gamma function.